be/src/storage/memtable.cpp

```diff
@@ -219,6 +219,9 @@ Status MemTable::flush() {
     if (UNLIKELY(_result_chunk == nullptr)) {
         return Status::OK();
     }
+    if (_result_chunk->reach_capacity_limit()) {
+        return Status::InternalError("memtable size exceed the limit");
```

Comment:
Add more information for user, e.g. current size, limit, row number, column number

Reply:
also the tablet_id

Reply:
Fixed